setup.py

```diff
@@ -450,6 +450,7 @@ def run(self):
     cmdclass={"build": build, "test": test},
     packages=packages,
     package_data={"gramps": package_data},
+    install_requires=["attrs"],
```

Comment:
Is this needed given your comment [above](https://github.com/gramps-project/gramps/pull/1837/files#r1901429036)?
Personally I'd still prefer to see `attrs.converters.to_bool()` used!

Reply:
No it isn't needed; I'll fix. If you can figure out how to add requirements to gramps, I'll let you change it, either here or in a follow-up PR.

Reply:
Let's work on the basis of a separate PR given the planned release schedule. Then if I get anything done before this PR is merged, it's a bonus.

Reply:
Sounds good.